lib/hooks/useList.ts

```diff
@@ -69,6 +70,36 @@ const useFetchListContributors = (id: string, range = 30) => {
   };
 };
 
+const addListContributor = async (listId: string, contributors: number[]) => {
+  const sessionResponse = await supabase.auth.getSession();
+  const sessionToken = sessionResponse?.data.session?.access_token;
+
+  try {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/lists/${listId}/contributors`, {
+      method: "POST",
+      body: JSON.stringify({ contributors }),
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${sessionToken}`,
+      },
+    });
+
+    if (response.ok) {
+      const data = await response.json();
+      return {
+        data,
+        error: null,
+      };
+    }
```

Comment:
Needs another return in case the response is not `ok` but doesn't throw an error